DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -66,17 +66,17 @@ public IEnumerable<CourseStatistics> GetCentreSpecificCourseStatistics(int centr
         )
         {
             return courseDataService.GetDelegateCoursesInfo(delegateId)
+                .Where(info => info.CustomisationCentreId == centreId)
                 .Where(info => courseCategoryId == null || info.CourseCategoryId == courseCategoryId)

```

Comment:
Could these Wheres be combined, or do you think it reads better as is?

Reply:
yeah they could be, but I prefer to keep the separate ones in different statements for readability. This would be the following if combined:
`.Where(info => info.CustomisationCentreId == centreId && (courseCategoryId == null || info.CourseCategoryId == courseCategoryId))`

Reply:
That's fair, I agree, it can be left as it is.